### PR TITLE
remove plugin import command from yarn2 install

### PR DESCRIPTION
**Pull request by arroyc**
- [X] The purpose of this PR is explained in this message or in an issue. 
- [X] Tests are included and/or updated for code changes.
- [X] Proper license headers are included in each file.

### CODE REVIEW

File: tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs [hunk 1]
```diff
@@ -976,6 +976,9 @@ public void  CanBuildAndRunNodeApp_UsingYarn2ForBuild()
             var appOutputDir = "/tmp/nextjs-yarn2-example";
             var appDir = volume.ContainerDir;
             var script = new ShellScriptBuilder()
+                .AddCommand($"yarn set version berry")
```
Comment (arroyc) [inline tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsOtherTests.cs:979]: we needed to set the workspace one time, this should  be user's responsibility, so adding these commands in test prior to oryx build command.
Assumption is User will run these commands in his/her local env before checking it in (including workspace) in git

File: src/BuildScriptGenerator/Node/NodeConstants.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public static class NodeConstants
         public const string YarnVersionCommand = "echo Using Yarn version: && yarn --version";
         public const string HugoVersionCommand = "echo Using Hugo version: && hugo version";
         public const string YarnPackageInstallCommand = "yarn install --prefer-offline";
-        public const string Yarn2PackageInstallCommand = "yarn plugin import workspace-tools && yarn workspaces focus --all";
+        public const string Yarn2PackageInstallCommand = "yarn workspaces focus --all";
```
Comment (iarna) [inline src/BuildScriptGenerator/Node/NodeConstants.cs:37]: Doesn't this mean that users without workspaces will fail? You can call yarn install if the workspace focus subcommand is unavailable with:

```
yarn workspaces focus --all || yarn install
```

Comment (arroyc) [reply src/BuildScriptGenerator/Node/NodeConstants.cs:37]: yeah  good catch .. i will add it ...

**Review by iarna — APPROVED** [review]

File: src/BuildScriptGenerator/Node/NodeConstants.cs [hunk 3]
```diff
@@ -34,7 +34,7 @@ public static class NodeConstants
         public const string YarnVersionCommand = "echo Using Yarn version: && yarn --version";
         public const string HugoVersionCommand = "echo Using Hugo version: && hugo version";
         public const string YarnPackageInstallCommand = "yarn install --prefer-offline";
-        public const string Yarn2PackageInstallCommand = "yarn plugin import workspace-tools && yarn workspaces focus --all";
+        public const string Yarn2PackageInstallCommand = "yarn workspaces focus --all || yarn install";
```
Comment (qianz2) [inline src/BuildScriptGenerator/Node/NodeConstants.cs:37]: `yarn plugin import workspace-tools` I think this is the plugin required for running `yarn workspaces focus --all`, if we remove it we may not able to run the later command.

Comment (arroyc) [reply src/BuildScriptGenerator/Node/NodeConstants.cs:37]: @qianz2  ... so basicaly you are kind of true but the whole scenario/workflow is little different ... here is a good reference https://github.com/yarnpkg/berry/issues/3199

so based on this, I think "yarn plugin import workspace-tools" is usually gets executed only once, it should not run every time .. user should run it once and check the plugin in in their repo and from next time onwards it should be only yarn workspace focus --all (as the workspace is already been set). So assumption is this command is basically user's responsibility and oryx should not do it.

If the workspace is not set yarn workspace focus --all will fail but then we will run yarn install command for yarn2.

Comment (arroyc) [reply src/BuildScriptGenerator/Node/NodeConstants.cs:37]: @qianz2  you can look at the test that I added to get more insight ..  let me know if you have some questions

Comment (qianz2) [reply src/BuildScriptGenerator/Node/NodeConstants.cs:37]: I see, then we will let users run this plugin command, thanks for the explain. And we can include this into oryx docs.

**Review by qianz2 — APPROVED** [review]

**Pull request merged**